src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -46,13 +46,18 @@ export const buildATIPageTrackPath = ({
   const referrer = getReferrer(platform, origin, previousPath);
   const campaignType = getCampaignType();
 
+  // on AMP, variable substitutions are used in the value and they cannot be
+  // encoded: https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md
+  const disableEncodingDueToAmpSubstitution = platform === 'amp';
+
   // We use amp variable substitutes to get the href and referrer and these cannot be manipulated
   // For canonical, we have a requirement to encode the x5 and x6 value twice. Source issue: https://github.com/bbc/simorgh/pull/6593
-  const x5Value = platform === 'amp' ? href : href && encodeURIComponent(href);
-  const x6Value =
-    platform === 'amp'
-      ? referrer
-      : referrer && encodeURIComponent(encodeURIComponent(referrer));
+  const x5Value = disableEncodingDueToAmpSubstitution
+    ? href
+    : href && encodeURIComponent(encodeURIComponent(href));
+  const x6Value = disableEncodingDueToAmpSubstitution
+    ? referrer
+    : referrer && encodeURIComponent(encodeURIComponent(referrer));
```

Comment:
Per my comment https://github.com/bbc/simorgh/pull/7854#discussion_r492602712 - could we check for null values within `encodeURIComponent`?

Reply:
I just checked this in dev tools:
![image](https://user-images.githubusercontent.com/9645462/93873785-391cbf00-fcca-11ea-97ce-bc58c80e3478.png)

Passing in null to `encodeURIComponent` gives the string `'null'`, I don't think we want that? The current code would return `null` as the value and the `x6` parameter would be omitted from the ATI hit which is correct behaviour.

Reply:
Forgot encodeURIComponent was a built-in function. Thanks.